config/locales/en/admin.yml

```diff
@@ -621,6 +621,16 @@ en:
         add: Add
       search:
         title: "Managers: User search"
+    sdg_managers:
+      index:
+        title: SDG Managers
+        name: Name
+        email: Email
+        no_sdg_managers: There are no SDG Managers.
+      sdg_manager:
+        add: Add
```

Comment:
There's a missing translation error here :thinking:. The expected key is `admin.sdg/managers.sdg/manager.add`.